Compilation with RiVLib fails

Commit 83bf2dccaf1 breaks compilation with RiVLib for me. I don't understand the build process well enough to fix it. CMake warns that linked items are dropped.
```
WARNING: Target "rxptest" requests linking to directory "/home/stefan/Documents/Uni/Heidelberg/Masterarbeit/05_processing/01_preprocessing/tools/pdal_compilation/rivlib-2_5_7-x86_64-linux-gcc55/include".  Targets may link only to libraries.  CMake is dropping the item.
WARNING: Target "pdal_plugin_reader_rxp" requests linking to directory "/home/stefan/Documents/Uni/Heidelberg/Masterarbeit/05_processing/01_preprocessing/tools/pdal_compilation/rivlib-2_5_7-x86_64-linux-gcc55/include".  Targets may link only to libraries.  CMake is dropping the item.
```
This results in the fatal error ```plugins/rxp/io/RxpPointcloud.hpp:48:10: fatal error: riegl/scanlib.hpp: No such file or directory```

@abellgithub Could you have a look, please?

--- a/plugins/rxp/CMakeLists.txt
+++ b/plugins/rxp/CMakeLists.txt
@@ -29,6 +29,7 @@ if (BUILD_RIVLIB_TESTS)
         LINK_WITH
             ${libname}
         INCLUDES
+            ${RiVLib_INCLUDE_DIRS}
             ${PROJECT_BINARY_DIR}/plugins/rxp/test
             ${PROJECT_SOURCE_DIR}/plugins/rxp/io
     )

